code/game/objects/items/documents.dm

```diff
@@ -115,3 +221,218 @@
 		. += "<span class='notice'>Wait a minute, this isn't an encrypted inspection report! You should throw it away.</span>"
 	else
 		. += "<span class='notice'>Wait a minute, this thing's blank! You should throw it away.</span>"
+
+/**
+ * # Fake N-spect scanner
+ *
+ * A clown variant of the N-spect scanner
+ *
+ * This prints fake reports with garbage in them,
+ * can be set to print them instantly with a screwdriver.
+ * By default it plays the old "woody" scanning sound, scanning sounds can be cycled by clicking with a multitool.
+ * Can be crafted into a bananium HONK-spect scanner
+ */
+/obj/item/inspector/clown
+	///will only cycle through modes with numbers lower than this
+	var/max_mode = CLOWN_INSPECTOR_PRINT_SOUND_MODE_LAST
+	///names of modes, ordered first to last
+	var/list/mode_names = list("normal", "classic", "honk", "fafafoggy", "fafafoggy (varied)")
+
+/obj/item/inspector/clown/attack(mob/living/M, mob/living/user)
+	. = ..()
+	print_report(user)
+
+/obj/item/inspector/clown/attackby(obj/item/I, mob/user, params)
+	if(cell_cover_open)
+		if(istype(I, /obj/item/kitchen/fork))
+			cycle_sound(user)
+		else if(I.tool_behaviour == TOOL_SCREWDRIVER)
+			cycle_print_time(user)
+	return ..()
+
+/obj/item/inspector/clown/examine(mob/user)
+	. = ..()
+	if(cell_cover_open)
+		. += "Two weird settings dials are visible within the battery compartment."
+
+/obj/item/inspector/clown/examine_more(mob/user)
+	if(cell_cover_open)
+		var/list/msg =  list("<span class='notice'>Both setting dials are flush with the surface of the battery compartment, and seem to be impossible to move with bare hands.</span>")
+		msg += "\t<span class='info'>The first dial is labeled \"SPEED\" and looks a bit like a <strong>screw</strong> head.</span>"
+		msg += "\t<span class='info'>The second dial is labeled \"SOUND\". It has four small holes in it. Perhaps it can be turned with a fork?</span>"
+		return msg
+	return ..()
```

Comment:
```suggestion
	if(!cell_cover_open)
		return ..()
	. = list("<span class='notice'>Both setting dials are flush with the surface of the battery compartment, and seem to be impossible to move with bare hands.</span>")
	. += "\t<span class='info'>The first dial is labeled \"SPEED\" and looks a bit like a <strong>screw</strong> head.</span>"
	. += "\t<span class='info'>The second dial is labeled \"SOUND\". It has four small holes in it. Perhaps it can be turned with a fork?</span>"

```
* Early return.